### DISCUSSION TITLE
Room Edit

### DISCUSSION THREAD

**Discussion by pophitru** [Tasshack/dreame-vacuum#186] [Q&A] [answered]
Just installed the release, works like a charm!
Got a question - how can I manually divide all the floor space to rooms/zones as I want (not as the Mi app made)?

**Comment by Tasshack** [comment, marked as answer]
Unfortunately this integration cannot provide a feature that app does not have.
If you are planning to use vacuum map card, you can use its [predefined options feature](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card#predefined-selection-options) combined with `vacuum_clean_zone` service of this integration.
